src/renderers/webgl/objects/label/utils.ts

```diff
@@ -0,0 +1,249 @@
+import { MIN_ZOOM } from '../..'
+import type { Stroke } from '../../../../types'
+import {
+  Text,
+  TextStyle,
+  TextStyleFill,
+  ITextStyle,
+  IBitmapTextStyle,
+  TextStyleAlign,
+  BitmapFont,
+  ColorSource,
+  TextStyleFontWeight,
+  LINE_JOIN
+} from 'pixi.js'
+
+export type LabelPosition = 'bottom' | 'left' | 'top' | 'right'
+
+export type LabelBackgroundStyle = {
+  color: ColorSource
```

Comment:
super minor, we should keep types and utils decoupled from pixi. i.e. this should be `color: string | number`, and we shouldn't expose types like TextStyleFill, TextStyleFontWeight. otherwise, it'll be harder to implement non-pixi renderers, or if a library just needs to import trellis types, they'd still have to import all of pixi.